item/Yonael; Firearms of the Realm.json

```diff
@@ -1,743 +1,1865 @@
+                                 ]
+                              },
+                              {
+                                 "type": "entries",
+                                 "name": "Smokepowder.",
+                                 "entries": [
+                                    "Smokepowder functions as Gunpowder does, as detailed on page 267 of the Dungeon Master's Guide, except as follows: Smokepowder is an Uncommon wondrous item, it explodes if set on fire, dropped, or otherwise handled roughly, it deals the listed damage in a 20-foot radius when it explodes, setting fire to an ounce of it causes a small explosion rather than a slow burn, and casting dispel magic on it renders it permanently inert."
+                                 ]
+                              },
+                              {
+                                 "type": "entries",
+                                 "name": "Foeshredder.",
+                                 "entries": [
+                                    "A foeshredder is a specialized grenade with an attached handle, allowing it to be thrown twice as far. A foeshredder functions identically to a grenade except that it can be thrown up to 120 feet away."
+                                 ]
+                              },
+                              {
+                                 "type": "entries",
+                                 "name": "Smokebomb.",
+                                 "entries": [
+                                    "A smokebomb consists of a wooden tube filled with alchemical compounds and smokepowder. As an action, a creature can light a smokebomb and throw it at a point up to 60 feet away. One round after a smokebomb lands, it emits a cloud of smoke that creates a heavily obscured area in a 20-foot radius. A moderate wind (at least 10 miles per hour) disperses the smoke in 4 rounds; a strong wind (20 or more miles per hour) disperses it in 1 round."
+                                 ]
+                              }
+                           ]
+                        }
+                     ]
+                  },
+                  {
+                     "type": "entries",
+                     "name": "Siege Equipment",
+                     "page": 3,
+                     "entries": [
+                        "Besides {@object Cannon|DMG|Cannons} (DMG, pg 255) employed by the wizards of Thay and the pirates of the Pirate Isles, the world of Faerûn is also home to organ guns, AKA ribalds."
+                     ]
+                  },
+				   "//TODO add ribald properly"
```

Comment:
iirc JSON doesn't like random tab-indented lines in the middle of space-indented lines, so this is probably killing things